manifests/e/ExacqTechnologies/exacqVisionClient/22.12.2.0/ExacqTechnologies.exacqVisionClient.installer.yaml

```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: ExacqTechnologies.exacqVisionClient
+PackageVersion: 22.12.2.0
+Installers:
+- InstallerUrl: https://cdnpublic.exacq.com/22.12/exacqVisionClient_22.12.2.0_x64.msi
+  InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: msi
+  InstallerSha256: A9FFCA61C34E77DF9436BB7036BB71CB8DE427D8543E74F5D7422F665725126C
+  ProductCode: '{133D0A53-AABC-48AE-98DA-B85512C51E78}'

```

Comment:
```suggestion
  ProductCode: '{133D0A53-AABC-48AE-98DA-B85512C51E78}'
  InstallerSwitches:
    Custom: /quiet
```

Reply:
Using `/S` gives the installer switch menu:
![image](https://user-images.githubusercontent.com/114952565/225755105-63b62182-1e1a-438f-8ef6-7b2beb372aa4.png)